Tests/TestingTests/ExitTestTests.swift

```diff
@@ -439,7 +439,7 @@ private import _TestingInternals
 }
 
 #if false // intentionally fails to compile
-@Test(arguments: 100 ..< 200)
```

Comment:
To be clear, this test doesn't ever compile. It's useful when testing exit tests to verify what happens if you attempt to close over some state.

Reply:
Yeah, understood. Despite being compiled out, without this trait it can still be included in the indexer's test discovery.